src/style.scss

```diff
@@ -27,27 +27,35 @@ a:hover {
   padding-bottom: 0.7rem;
 }
 
-.my-15 {
+.my15 {
   margin-top: 15rem;
   margin-bottom: 15rem;
 }
```

Comment:
Bootstrap documentation. 

> (You can add more sizes by adding entries to the $spacers Sass map variable.)

Reply:
Just random thought. 

I personally prefer to keep all "utilities" class names to be consistent with bootstrap style. (e.g. `line-height-sm`, `line-height-lg`, and `letter-spacing-md`...)

And use your camelSpace style for the semantic structures only. (e.g. `readMore` and `sectionHeader`)